### PR TITLE
DCS-118: User is now able to view the current statements before movin…

### PR DESCRIPTION
**Pull request by GurnankCheema**
…g to an Add Comments page

http://localhost:3000/reportId/add-comment-to-statement

The previous journey was for user to view and amend the Statement on the same page.
The button to access the Statement page was on the Incidents page
The journey has been changed so that the the Incidents page only has a view button. This takes user to the Statement page which displays the initial statement plus any additional comments.
From here the user can either click Continue to go back to the Incidents page or click Add comment to go to the new page.
The new page displays the same information as the Statement page but also has a text box to accept another comment.
The user then has the option to either Save a new comment (and go back to the Incidents page) or Cancel and return to the previous page.

### CODE REVIEW

File: server/routes/incidents.js [hunk 1]
```diff
@@ -151,7 +151,27 @@ module.exports = function CreateReportRoutes({ statementService, offenderService
       if (req.body.additionalComment && req.body.additionalComment.trim().length) {
         await statementService.saveAdditionalComment(statement.id, req.body.additionalComment)
       }
-      return res.redirect(`/${reportId}/your-statement`)
+      return res.redirect(`/`)
+    },
+
+    addCommentToStatement: async (req, res) => {
```
Comment (andrewrlee) [inline server/routes/incidents.js]: can we call this `viewAddCommentToStatement` so it matches the same convention as the other routes?

File: server/routes/incidents.test.js [hunk 2]
```diff
@@ -110,13 +110,13 @@ describe('GET /:reportId/your-statement', () => {
       }))
 })
 
-describe('POST /:reportId/your-statement', () => {
-  it('should save ammendment', () =>
+describe('POST /:reportId/add-comment-to-statement', () => {
```
Comment (andrewrlee) [inline server/routes/incidents.test.js]: think you need a test for the  `GET` add-comment-to-statement

File: server/views/pages/statement/your-statement.html [hunk 3]
```diff
@@ -55,69 +54,49 @@ <h1 class="govuk-heading-xl mainHeading">Your use of force statement</h1>
   </div>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0 govuk-label--xs statement" data-qa="statement">{{ data.statement }}  </p>
+      <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0 govuk-label--xs statement" data-qa="statement">
+        {{ data.statement }} </p>
```
Comment (andrewrlee) [inline server/views/pages/statement/your-statement.html]: think this is going to add extra padding to the start of the statement as it has `white-space: pre-wrap` - needs to be on one line

File: server/views/pages/statement/add-comment-to-statement.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    <div class="govuk-grid-column-one-half ">
+      <p>
+        <span class="govuk-label--s">Submitted:</span> {{ data.submittedDate | formatDate('D MMMM YYYY - HH:mm') }}
+      </p>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0 govuk-label--xs statement" data-qa="statement">
+        {{ data.statement }} </p>
```
Comment (andrewrlee) [inline server/views/pages/statement/add-comment-to-statement.html]: hink this is going to add extra padding to the start of the statement as it has white-space: pre-wrap - needs to be on one line

File: server/routes/incidents.js (hunk 1 above)
Comment (GurnankCheema) [reply server/routes/incidents.js]: yes

File: server/views/pages/statement/your-statement.html (hunk 3 above)
Comment (GurnankCheema) [reply server/views/pages/statement/your-statement.html]: Yes, that happened when I used the linter. 
Amended

File: server/views/pages/statement/add-comment-to-statement.html (hunk 4 above)
Comment (GurnankCheema) [reply server/views/pages/statement/add-comment-to-statement.html]: as above.

File: server/routes/incidents.test.js (hunk 2 above)
Comment (GurnankCheema) [reply server/routes/incidents.test.js]: done

**Pull request merged**